index.html

```diff
@@ -13,53 +13,50 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <nav class="desktop-nav">
+    <div class="social-media">
+      <ul class="links">
+        <li><i class="fa fa-facebook"></i></li>
+        <li><i class="fa fa-twitter"></i></li>
+        <li>Logout</li>
+        <li>My page</li>
+        <li>korean</li>
+      </ul>
+    </div>
+    <header class="desktop-nav">
       <div class="logo">
-        <a href="index.html"> <img src="image/logo5.png" alt="" /></a>
-      </div>
-
-      <div class="nav-desktop-links">
-        <ul>
-          <li><a href="about.html">About</a></li>
-          <li><a href="#">Feature</a></li>
-          <li><a href="#">Register</a></li>
-        </ul>
+        <a href="index.html"> <img src="image/logo77.png" alt="image" /></a>
       </div>
       <div class="toggle">
-        <i class="fa fa-bars"></i>
+      <i class="fa fa-bars"></i>
       </div>
-      <div class="toggle">
-        <i class="fa fa-xmark"></i>
+      <div class="toggle-x"> 
+           <button type="button">&times;</button>
       </div>
-    </nav>
-    <div class="nav-mobile-links">
+   
+    <nav class="nav-mobile-links">
       <ul>
+        <li class="home-menu"><a href="index.html">Home</a></li>
         <li><a href="about.html">About</a></li>
-        <li><a href="#">Feature</a></li>
+      
         <li><a href="#">Register</a></li>
+        <li class="cc-campaing"><a href="#">CC Campaing</a></li>
       </ul>
```

Comment:
- [ ] `required`
- [ ] Like the previous reviewer mentioned please move the logo and navbar like the original template
- [ ] Also please make sure you follow the fonts,size, and typography just as the original template
- [ ] please see screenshots below and compare them

`your template`
![image](https://user-images.githubusercontent.com/66230306/227712149-23fb5180-435a-4209-9b82-95e5c1f1e762.png)

`template to follow`
![image](https://user-images.githubusercontent.com/66230306/227712268-63dd24ba-4d39-492c-be89-267d76b94f1f.png)